### ISSUE TITLE
Google Service 20.12.15 does not work

### ISSUE THREAD

**Issue by Creatik** [Eselter/AA-Phenotype-Patcher#7]
Hey. Starting with version 20.12.15 (Google service). The patch does not work.

**Comment by hugodigio** [comment]
I confirm. need to downgrade google play services to work again. Do you know alternative patcher that work with last google services updates ?

**Comment by Creatik** [comment]
Yes, the problem was beta testing Google services. You need to exit the testers, roll back Google services and update.
